kev/backends/s3/db.py

```diff
@@ -96,7 +96,7 @@ def add_indexes(self, doc_obj, doc):
     def evaluate(self, filters_list, doc_class):
         if len(filters_list) == 1:
             filter_value = filters_list[0]
-            id_list = self._indexer.objects.filter(Prefix=filters_list[0])
+            id_list = self._indexer.objects.filter(Prefix=filter_value)
```

Comment:
filter_value was unused and I was performing some code cleanup. It seemed the original intention was to provide the first filter item to objects.filter().

Either I could have removed 'filter_value = filters_list[0]' or replaced 'filters_list[0]' with 'filter_value' since they are equal. I went with the latter. If you prefer, I can revert the change and just remove 'filter_value = filters_list[0]'.